cmd/admin-heal-ops.go

```diff
@@ -876,9 +876,9 @@ func (h *healSequence) healBucket(bucket string, bucketsOnly bool) error {
 		if h.object != "" {
 			// Check if an object named as the objPrefix exists,
 			// and if so heal it.
-			_, err := objectAPI.GetObjectInfo(h.ctx, bucket, h.object, ObjectOptions{})
+			oi, err := objectAPI.GetObjectInfo(h.ctx, bucket, h.object, ObjectOptions{})
 			if err == nil {
-				if err = h.healObject(bucket, h.object, ""); err != nil {
+				if err = h.healObject(bucket, h.object, oi.VersionID); err != nil {
```

Comment:
NOTE: since `ObjectOptions{}` didn't specify any version, oi.VersionID is latest version.

Reply:
I am not sure if this code changes any behavior, is something not working @klauspost ?

Reply:
@vadmeste According to Harsha the version ID is required for it to operate consistently.

It may be code that is never active, in which case it should just be deleted.